src/Event.tsx

```diff
@@ -60,7 +60,7 @@ class Event extends React.Component<EventProps, EventState> {
       )
     this.setState({
       event: event,
-      userHasLogAccess: authzList.items.length > 0
+      userHasLogAccess: authzList.items && authzList.items.length > 0
```

Comment:
@krancour I believe that check for authzList.items is enough and there is no need for checking if length > 0. Can you explain the reason for the second check ?

Reply:
Because an empty array isn't falsy.